core/src/main/scala/com/armanbilge/scalanative/brew/Brew.scala

```diff
@@ -21,7 +21,7 @@ import io.circe.jawn
 
 import java.nio.file.Path
 import java.nio.file.Paths
-import scala.scalanative.build.Platform
```

Comment:
`Platform` is package-private since 0.5.0: https://github.com/scala-native/scala-native/blob/v0.5.0/tools/jvm/src/main/scala/scala/scalanative/build/Platform.scala.